src/windows/list.cpp

```diff
@@ -148,21 +148,30 @@ bool IsVirtualHardDrive(HDEVINFO hDeviceInfo, SP_DEVINFO_DATA deviceInfoData) {
   return false;
 }
 
-bool IsSystemDevice(HDEVINFO hDeviceInfo, SP_DEVINFO_DATA deviceInfoData) {
+bool IsSystemDevice(HDEVINFO hDeviceInfo, DeviceDescriptor *device) {
   PWSTR windowsPath = NULL;
```

Comment:
Minor nit, but perhaps this variable should be renamed to `folderPath`? (and should its declaration be moved to inside the `for` loop?)

Reply:
Done